publish/run-all-rpms.sh

```diff
@@ -40,12 +40,14 @@ while true; do
   esac
 
   for conf in "$@"; do
-    "$aliPublish" --config "$conf" --debug sync-rpms >&2
+    if "$aliPublish" --config "$conf" --debug sync-rpms >&2; then
```

Comment:
What if the upload fails? will it retry?

Reply:
No, if `rclone` fails, it'll only retry on the next iteration.